app/models/endpoint.rb

```diff
@@ -24,4 +50,18 @@ def resolve_verify_ssl_value(val)
     else            val
     end
   end
+
+  # Returns a list, to support concatenated PEM certs.
+  def parse_certificate_authority
+    return [] if certificate_authority.blank?
+    certificate_authority.split(/(?=-----BEGIN)/).reject(&:blank?).collect do |pem_fragment|
```

Comment:
Yes, at the end of endpoint_spec.rb: https://github.com/ManageIQ/manageiq/pull/13567/files#diff-763829b60a325924ed156fcba4a726e4R123
(in API test I didn't bother)